c2cgeoportal/views/pdfreport.py

```diff
@@ -155,7 +157,7 @@ def get_report(self):
                 "outputformat": "gml3",
                 "request": "GetFeature",
                 "typeName": self.layername,
-                "featureid": self.layername + "." + id,
+                "featureid": ",".join(features_ids),
```

Comment:
Isn't the role of who write the report that there is no possible injection?
Because we can use the attribute just to display it...